fwutil/lib.py

```diff
@@ -790,3 +1020,39 @@ def get_status(self):
                         append_module_name = False
 
         return tabulate(status_table, self.HEADER, tablefmt=self.FORMAT)
+
+    def read_au_status_file_if_exists(self, filename=FW_AU_STATUS_FILE_PATH):
+        data = None
+        if os.path.exists(filename):
+            with open(filename) as au_status_file:
+                data = json.load(au_status_file)
+        return data
+
+    def get_au_status(self):
+        auto_updated_status_table = []
+        data = self.read_au_status_file_if_exists(FW_AU_STATUS_FILE_PATH)
+
+        if data is None:
+            return None
+
+        for comp_path, comp_au_status in data.items():
+            if not self.__is_dict(comp_au_status):
```

Comment:
Since the value for comp_path is a list, please iterate over it seperately.

```suggestion
        for comp_path, au_status in data.items():
            for comp_au_status in au_status:
                if not self.__is_dict(comp_au_status):
```